dashboard/app/dsl/content_dsl.rb

```diff
@@ -6,6 +6,7 @@ def initialize
   end
   def title(text) @hash[:title] = text end
   def type(text) @hash[:type] = text end
+  def assessment(text) @hash[:assessment] = text end
```

Comment:
not sure whether this is best done in the .multi or in the .script (as is already done with the "other" assessments)